server/plugin/api.go

```diff
@@ -90,22 +83,20 @@ func (p *MatterpollPlugin) handleVote(w http.ResponseWriter, r *http.Request) {
 	}
 
 	hasVoted := poll.HasVoted(userID)
-	err := poll.UpdateVote(userID, optionNumber)
-	if err != nil {
+	if err := poll.UpdateVote(userID, optionNumber); err != nil {
```

Comment:
Should we avoid idiomic error handling?  `if err := method(); err != nil { ... }`

Reply:
I'm fine using `if err := method(); err != nil { ... }`. You might also want to take a look at https://developers.mattermost.com/blog/idiomatic-error-handling.

Reply:
I got it. You're right. There is no need to fix this.